### DISCUSSION TITLE
Issues deploying site

### DISCUSSION THREAD

**Discussion by Andre601** [vercel/vercel#4016] [General] [answered]
I face a weird issue, where the now bot comments the following message on commits I make:

> Deployment failed with the following error:
>
> ```
> You defined 1 build that did not match any source files (please ensure they are NOT defined in .nowignore):
> ```

I don't have any `.nowignore` file defined and the files I edited where located under `override` in the main (root) directory of the repository.

The bot itself is configured to use the root of the repository, to then take the built content from the `site` directory.
I have a `now.json` with the following settings:
```json
{
  "github": {
    "silent": true
  }
}
```

**Comment by styfle** [comment]
Hi @Andre601 its difficult to know if this is an issue with our platform or something is misconfigured in your project.

Can you share the source code?

If not, can you contact our team through this support form? https://zeit.co/support/request

**Reply by Andre601** [reply]
https://github.com/PowerPlugins/Plugin-Docs

Commit message of the bot:
https://github.com/PowerPlugins/Plugin-Docs/commit/10654c6b3f738cc299f67b0d46edf373a0bdaaa6#comments

**Comment by Andre601** [comment]
It sometimes even posts twice
![image](https://user-images.githubusercontent.com/11576465/78997799-94595b80-7b47-11ea-8505-f31aa9941da6.png)

**Comment by styfle** [comment, marked as answer]
Thanks for [the repo](https://github.com/PowerPlugins/Plugin-Docs)!

This is indeed a bug in our zero-config detection. It is trying to use `now.json` as the source file.

I created PR #4042 to fix this. The workaround is to delete `now.json`.

**Reply by Andre601** [reply]
Thanks, but the issue would then be that the bot comments on every commit I make which I try to avoid, hence the now.json file with the `github.silent` setting set to true.

**Reply by styfle** [reply]
The fix is on the canary channel and will be available in the next stable release, thanks!
